client/components/App.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <>
```

Comment:
Nice work with the fragments. I feel like this will give us some more room when we go to integrate our widgets and add css.